app/src/features/FhirResourceTree/TreeItem.tsx

```diff
@@ -41,25 +41,26 @@ const TreeItem = ({
   hasParentExpanded,
 }: TreeItemProps): JSX.Element => {
   const classes = useStyle();
-
   const isNodeHidden =
     elementNode.type === "Extension" &&
     elementNode.isArray &&
     elementNode.children.length === 0;
   const [hasExpanded, setHasExpanded] = useState(false);
-  const isPrimitive = elementNode.kind === "primitive";
   const isComplex = elementNode.kind === "complex";
+  const hasToFetchNodeDefinition =
+    isComplex &&
+    hasParentExpanded &&
+    !elementNode.isArray &&
+    elementNode.definitionNode.childrenDefinitions.filter(
+      ({ definition }) => !definition.sliceName
+    ).length === 0;
```

Comment:
can you name this part `elementNode.definitionNode.childrenDefinitions.filter(
      ({ definition }) => !definition.sliceName
    ).length === 0` ?